Resources/Prototypes/Loadouts/head.yml

```diff
@@ -70,6 +70,14 @@
   exclusive: true
   items:
     - ClothingHeadHatBellhop
+
+- type: loadout
+  id: LoadoutHeadFlower
+  category: Head
+  cost: 2
```

Comment:
Does it have to cost "2" points just for a flower?
1 seem fine.

Reply:
lowered the cost to 1 supposing I did this right, should be good